src/r5py/util/memory_footprint.py

```diff
@@ -49,14 +49,14 @@ def _share_of_ram(share=0.8, leave_at_least=(2 * (2**10))):
 
     Returns
     -------
-    float
+    int
         A value in MiB that is close to `share` portion of total RAM.
     """
     total_ram = psutil.virtual_memory().total / (2**20)
     if total_ram * (1.0 - share) > leave_at_least:
-        share_of_ram = round(share * total_ram)
+        share_of_ram = int(round(share * total_ram))
```

Comment:
`round()` returns `int`

Reply:
Ha, true. `np.round()` then again returns float, which to me is actually more logical. Was surprised learning that the Python's own round() indeed work like this (i.e. returns integer).

Reply:
It is inconsistent, in a way, because if called with a second argument (number of decimal places), it returns float ;)